components/switch/seeedmultichannelrelay.rst

```diff
@@ -0,0 +1,69 @@
+.. _seeedmultichannelrelay_switch:
+
+Seeed Multi Channel Relay Switch Component
+==========================================
+
+.. seo::
+    :description: Instructions for setting up seeedmultichannelrelay Switch.
+    :image: seeedmultichannelrelay.jpg
+
+The ``seeedmultichannelrelay`` switch platform supports the 8 channels of the relay unit. There are multi relays which are supported.
+The relays can be used to switch up to AC-250V or DC-110V. For more information: 
+
+Below are several links to supported relays
+https://wiki.seeedstudio.com/Grove-4-Channel_SPDT_Relay/
+https://wiki.seeedstudio.com/Grove-8-Channel_Solid_State_Relay/
+
+.. figure:: images/Seeed-4-Channel-SPDT-Relay.png
+    :align: center
+    :width: 80.0%
+
+.. figure:: images/Seeed-8-Channel-Solid-State-Relay.png
+    :align: center
+    :width: 80.0%
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    seeedmultichannelrelay:
+        id: SeeedMultiChannelRelay_ID
+        address: 0x11 # changes the address of the seeedmultichannelrelay only runs this once
+        # ...
+
+    switch:
+      - platform: seeedmultichannelrelay
+        id: relay_1
+        name: relay1
+        channel: 1
+        seeedmultichannelrelay_id: SeeedMultiChannelRelay_ID
+
+Configuration variables:
+------------------------
+**Control**
+
+- **addres** (*Optional*, hex-address): This should only be added to change the i2c address
```

Comment:
I think this name is not intuitive. How about to rename it to `change_address_to`?

I also miss info about the default address. This is from product page:
Default I2C Address: 0x11 or 0x12
Available I2C Address: 0x00 ~ 0x7F

Reply:
Added the default addresses and changed the name to change_address_to